runtime/include/vendor_version.h

```diff
@@ -32,3 +32,10 @@
  * #define VENDOR_SHORT_NAME "ABC"
  * #define VENDOR_SHA "1a2b3c4"
  */
+
+#ifndef vendor_version_h
+#define vendor_version_h
+
+#define VENDOR_VERSION_STRING "0.8.0-internal"
```

Comment:
This change seems unnecessary since the same value is found in j9cfg.h.ftl

Reply:
Removed it to following existing pattern, use the value in `j9cfg.h.ftl`.